requirements.txt

```diff
@@ -1 +1,2 @@
 six>=1.7.0
+monotonic>=0.6 # Apache-2.0
```

Comment:
I can, although how this package is coded it imports the py3 version and just uses that when it can.

https://github.com/atdt/monotonic/blob/master/monotonic.py#L51

We can do the same thing if we want, or just let the module there  do it.